src/components/DuosHeader.js

```diff
@@ -2,10 +1,111 @@
+import {Component, Fragment, useState} from 'react';
 import {a, button, div, h, hr, img, li, nav, small, span, ul} from 'react-hyperscript-helpers';
-import ResponsiveMenu from 'react-responsive-navbar';
+import Drawer from '@material-ui/core/Drawer';
+import Hidden from '@material-ui/core/Hidden';
+import {IconButton, List, ListItem, Menu, MenuItem} from '@material-ui/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6963768)
```suggestion
import { withStyles } from "@material-ui/core/styles";
```

Reply:
Pretty sure single quotes for strings is a JS standard. The vast majority of JS code uses single quotes for import statements. Heck, React's source code uses single-quotes for it's import statements. Also popular JS linters (ESLint in my case) and public rulesets (AirBnb, Facebook) are set up for single-quote strings.